frontend/src/components/search/Filters/CheckboxFilter.tsx

```diff
@@ -62,31 +53,13 @@ export function CheckboxFilterBody({
         )}
         {filterOptions.map((option) => (
           <li key={option.id}>
-            {/* If we have children, show a "section", otherwise show just a checkbox */}
-            {option.children ? (
```

Comment:
since the agency filter is the only thing that uses this nested setup, moved that into the agency filter body to clean this up for vanilla implementations